src/views/Sjofn/Practice42.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <div>
+    Practice 4-2
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Practice1',
+  data () {
+    return {
+      genres: [
+        'Horror', 'Detective', 'Adventure', 'Romance'
+      ],
+      disciplines: [
+        'Chemistry', 'Mathematics', 'Literature', 'English'
+      ],
+      livingRoomDesign: {
+        style: 'Minimalism',
+        shape: 'Square',
+        area: 20,
+        mainColor: 'beige',
+        accentColors: ['dark green', 'dark orange'],
+        windows: 3,
+        hasPanoramicView: true,
+        isParquet: true,
+        isFireplace: true,
+        ceilingsMinHeight: 5
+      },
+      fictionCharacter: [
+        {
+          gender: 'male',
+          age: 25,
+          firstName: 'Martin',
+          lastName: 'Corrigan',
+          author: 'James Hadley Chase',
+          country: 'United States',
+          language: 'English',
+          specialties: ['special agent', 'detective'],
+          isGood: true,
+          isSuccessful: true
+        }
+      ],
+      z: '',
+      a: '',
+      b: '',
+      m: '',
+      n: '',
+      l: '',
+      p: '',
+      min: '',
+      max: ''
```

Comment:
**Please, don't create a data-attribute for every value you use.**
Many of these are redundant, never used and should be removed.

For example, let's check 'z' value. 
It is an argument of `getPowFive` method. Everything you pass into the mthod transforms into local variable 'z'. The method doesn't require any dependencies from outer scope. Why you added 'z' in `data` then?
Please, remove it, check other and remove all redundant ones.

Reply:
Contact me if it is not clear.